src/app/utils/formValidationUtils.js

```diff
@@ -156,19 +142,19 @@ function validate(form, cb) {
       validate: () => true,
       errorMsg: '',
     },
-    'starting-page': {
-      validate: (val) => isNumeric(val),
-      errorMsg: 'Please enter the starting page',
+    startPage: {
+      validate: (val) => (isNumeric(val) && val > '0'),
+      errorMsg: 'Please enter the start page',
     },
-    'ending-page': {
-      validate: (val) => isNumeric(val),
+    endPage: {
+      validate: (val) => (isNumeric(val) && val > '0'),
```

Comment:
Cool. I prefer that `.toString` just because it's clearer but as long as we remember what it does, it's fine.